### DISCUSSION TITLE
What is the prediction accuracy of force by DeepH?

### DISCUSSION THREAD

**Discussion by chunweizhu** [mzjb/DeepH-pack#55] [Q&A]
Dear Dr. Li, 

I hope this message finds you well. I was wondering if I may ask you a few questions regarding the `DeepH` model. Firstly, could you kindly clarify what loss function is used in the model, and what is the prediction error of the deep neural network constructed by `DeepH` for the force acting on each atom in the system? Also, if the error is deemed small, may the trained network be used for molecular dynamics simulation akin to the `DeepMD-kit` ? Moreover, I am curious about the methodology employed to obtain the ground state structure; did you utilize perturbation techniques and energy predictions for the structure, or is there another approach employed for this purpose? 

Thank you in advance for your assistance.
